enterprise/frontend/src/metabase-enterprise/data_editing/tables/view/BrowseTableData.module.css

```diff
@@ -0,0 +1,16 @@
+.container {
+  height: 100%;
+  width: 100%;
+  overflow: hidden;
+}
+
+.gridWrapper {
+  overflow: hidden;
+  flex-grow: 1;
+}
+
+.gridRoot {
+  border: 1px solid var(--mb-color-border);
+  border-radius: 0.425rem;
```

Comment:
nit: consider using one of these https://mantine.dev/styles/css-variables/#border-radius-variables